ui/src/pages/home/apps-section/app-table/app-table.tsx

```diff
@@ -0,0 +1,169 @@
+            <TableBody>
+              {updatedApps.map((app) => (
+                <TableRow
+                  key={app.id}
+                  sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+                >
+                  <TableCell component="th" scope="row">
+                    {getIcon(app.public, app.shared)}
+                    <span className="inline relative icon-text">
+                      {app.name}
+                    </span>
+                  </TableCell>
+                  <TableCell>
+                    <StatusChip status={app.status} />
+                  </TableCell>
+                  <TableCell>{app.username}</TableCell>
+                  <TableCell>
+                    <Chip
+                      label={app.framework}
+                      variant="outlined"
+                      size="small"
+                    />
+                  </TableCell>
+                  <TableCell>
+                    {app.status === 'Stopped' || app.status === 'Ready' ? (
+                      <Button
+                        onClick={() => {
+                          setIsStartOpen(true);
+                          setCurrentApp(app);
+                        }}
+                        color="inherit"
+                        size="small"
+                        className="action-button"
+                        data-testid="PlayCircleRoundedIcon"
+                      >
+                        <PlayCircleRoundedIcon />
+                      </Button>
+                    ) : (
+                      <Button
+                        onClick={() => {
+                          setIsStopOpen(true);
+                          setCurrentApp(app);
+                        }}
+                        color="inherit"
+                        size="small"
+                        className="action-button"
+                        data-testid="StopCircleRoundedIcon"
+                      >
+                        <StopCircleRoundedIcon />
+                      </Button>
+                    )}
+                    <Button
+                      onClick={() =>
+                        (window.location.href = `${API_BASE_URL}/edit-app?id=${app.id}`)
+                      }
+                      color="inherit"
+                      size="small"
+                      className="action-button"
+                      data-testid="EditRoundedIcon"
+                    >
+                      <EditRoundedIcon />
+                    </Button>
+                    <Button
+                      onClick={() => {
+                        setIsDeleteOpen(true);
+                        setCurrentApp(app);
+                      }}
+                      color="inherit"
+                      size="small"
+                      className="action-button"
+                      data-testid="DeleteRoundedIcon"
+                    >
+                      <DeleteRoundedIcon />
+                    </Button>
```

Comment:
These buttons should be visible and/or disabled as follows:

- Stop Icon: if app.status === "Running"
-- Disable button if Shared
- Start Icon: show in all other cases
-- Disable if app.status === "Pending" or app.status === "Unknown"
- Edit Icon
-- Disable if Shared
- Delete Icon
-- Disable if Shared